docker-compose.yml

```diff
@@ -26,11 +23,6 @@ services:
       - "8080:8080"
     environment:
       - SERVER_PORT=8080
-      - DB_HOST=db
```

Comment:
we could still keep the default values here and use it in case .env is not present. use it like 
`- DB_HOST=${DB_HOST:-db}`